modules/osdk-cli-reference-run.adoc

```diff
@@ -7,54 +7,35 @@
 
 The `operator-sdk run` command provides options that can launch the Operator in various environments.
 
-.`run` arguments
+== packagemanifests
+`run packagemanifests` subcommand deploys an Operator's package manifests with OLM. The command argument must be set to a valid package manifest root directory, for example, ``<project-root>/packagemanifests`.
+
+.`packagemanifests` arguments
 [options="header",cols="1,3"]
 |===
 |Arguments |Description
 
+|`--include` (string)
+|The file path to Kubernetes resource manifests, for example, Role, Subscription. These supplement or override defaults generated by run/cleanup.
+
+|`--install-mode` (string)
+|The `OperatorGroup` is created with the specified `InstallMode`. Format: `InstallModeType[=ns1,ns2[, ...]]``
+
 |`--kubeconfig` (string)
-|The file path to a Kubernetes configuration file. Default: `$HOME/.kube/config`
+|The file path to a Kubernetes configuration file. Defaults to the location specified by `$KUBECONFIG`, or to default file rules if not set
```

Comment:
Please add a period to the end of the sentence.  Consider "...or to default file rules if the environment variable is not set."